isthmus/src/main/java/io/substrait/isthmus/expression/NestedExpressionConverter.java

```diff
@@ -0,0 +1,33 @@
+package io.substrait.isthmus.expression;
+
+import io.substrait.expression.Expression;
+import io.substrait.isthmus.CallConverter;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import org.apache.calcite.rex.RexCall;
+import org.apache.calcite.rex.RexNode;
+
+public class NestedExpressionConverter implements CallConverter {
+
+  public NestedExpressionConverter() {}
+
+  @Override
+  public Optional<Expression> convert(
+      RexCall call, Function<RexNode, Expression> topLevelConverter) {
+
+    if (!call.getOperator().getName().equals("NESTEDLIST")) {
```

Comment:
```suggestion
    if (!(call.getOperator() instanceof NestedListConstructor)) {
```

string-comparison is brittle and should be avoided